src/style.css

```
@@ -0,0 +1,38 @@
+body {
+  margin: unset;
+}
+
+.stars {
+  display: flex;
+}
+
+div .stars__star {
```

Необов'язкова специфічність
```suggestion
.stars__star {
```

---

done